index.html

```diff
@@ -61,264 +59,276 @@
-                                            <a href="https://github.com/" target="_blank">
-                                                <span class="fa fa-github"></span>
-                                            </a>
-                                        </li>
-                                    </ul>
-                                </div>
+                        <div class="about-us-name">
+                            <h3>Katarzyna Bęś-Laskowska</h3>
+                            <p> Lekarz weterynarii</p>
+                        </div>
+                    </div>
+                    <div class="about-us-inner-box">
+                        <div class="about-us-image">
+                            <img src="images/Kamil.jpg" alt="zdjęcie Kamila">
+                        </div>
+                        <div class="about-us-text">
+                            <div class="about-us-description">Slusznie nagrodzony tytułem "Najlepszy Prezes Fundacji
+                                2018." Dla Kamila nie ma rzeczy niemożliwych. Prywatnie posiada 5 psów i 13 kotów.
                             </div>
-                            <div class="about-us-name">
-                                <h3>Kamil Lubowicki</h3>
-                                <p>Prezes Zarządu</p>
+                            <div class="about-us-social-box">
+                                <ul class="social-icons">
+                                    <li>
+                                        <a href="https://www.facebook.com/" target="_blank">
+                                            <span class="fa fa-facebook-f"></span>
+                                        </a>
+                                    </li>
+                                    <li>
+                                        <a href="https://twitter.com/" target="_blank">
+                                            <span class="fa fa-twitter"></span>
+                                        </a>
+                                    </li>
+                                    <li>
+                                        <a href="https://github.com/" target="_blank">
+                                            <span class="fa fa-github"></span>
+                                        </a>
+                                    </li>
+                                </ul>
                             </div>
                         </div>
+                        <div class="about-us-name">
+                            <h3>Kamil Lubowicki</h3>
+                            <p>Prezes Zarządu</p>
+                        </div>
                     </div>
                 </div>
             </div>
-
         </div>
-
-
-
-
-        <div class="newsletter">
-
-
+    </div>
+    <div class="newsletter">
+        <img class="newsletter__image" src="images/newletter.png" alt="people-and-mails">
+        <div class="newsletter__content newsletter-content">
```

Comment:
hmm.. ja chyba bym sobie tu darował klasę `newsletter-content` natomiast, w tagach poniżej zamiast np. `newsletter-content__title` dałbym `newsletter__title` itd. bo tak w zasadzie to np. te `h2` jest tytułem tylko w tym newsletterze

Reply:
Pozmieniałam te klasy. Teraz powinno być przejrzyściej :)